actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/BulkUploadManagementActor.java

```diff
@@ -345,10 +345,10 @@ private void processBulkUserUpload(Map<String, Object> req, String processId) {
           ResponseCode.CLIENT_ERROR.getResponseCode());
     }
     if (null != userList) {
-      if (null != PropertiesCache.getInstance().getProperty(JsonKey.BULK_UPLOAD_USER_DATA_SIZE)) {
+      if (StringUtils.isNotBlank(ProjectUtil.getConfigValue(JsonKey.BULK_UPLOAD_USER_DATA_SIZE))) {
         userDataSize =
             (Integer.parseInt(
-                PropertiesCache.getInstance().getProperty(JsonKey.BULK_UPLOAD_USER_DATA_SIZE)));
+                ProjectUtil.getConfigValue(JsonKey.BULK_UPLOAD_USER_DATA_SIZE).trim()));
 
         ProjectLogger.log("bulk upload user data size read from config file " + userDataSize);
```

Comment:
change bulk upload user data size read from config file to "BulkUploadManagementActor:processBulkUserUpload  User upload config size: "  and make it info.